.github/workflows/CI.yml

```diff
@@ -264,100 +264,111 @@ jobs:
+                  "$regr_test-edge" \
+                  $REFERENCE/"$regr_test"/chemkin/chem_edge_annotated.inp \
+                  $REFERENCE/"$regr_test"/chemkin/species_edge_dictionary.txt \
+                  test/regression/"$regr_test"/chemkin/chem_edge_annotated.inp \
+                  test/regression/"$regr_test"/chemkin/species_edge_dictionary.txt
+              then
+                echo "$regr_test Passed Edge Comparison ✅"
+              else
+                echo "$regr_test Failed Edge Comparison ❌" | tee -a $GITHUB_STEP_SUMMARY
+                cp "$regr_test-edge.log" test/regression-diff/
+                export FAILED=Yes
+              fi
+              echo "</summary>"
+              cat "$regr_test-edge.log"
+              echo "</details>\n"
+
+              # Check for Regression between Reference and Dynamic (skip superminimal)
+              if [ -f test/regression/"$regr_test"/regression_input.py ];
+              then
+                echo "<details>"
+                if python-jl rmgpy/tools/regression.py \
+                  test/regression/"$regr_test"/regression_input.py \
+                  $REFERENCE/"$regr_test"/chemkin \
+                  test/regression/"$regr_test"/chemkin
+                then
+                  echo "<summary>$regr_test Passed Observable Testing ✅</summary>"
+                else
+                  echo "<summary>$regr_test Failed Observable Testing ❌</summary>" | tee -a $GITHUB_STEP_SUMMARY
+                  export FAILED=Yes
+                fi
+                echo "</details>"
+              fi
+              echo ""
+            done
+            if [[ ${FAILED} ]]; then
+              echo "⚠️ One or more regression tests failed." | tee -a $GITHUB_STEP_SUMMARY >&2
+              echo "Please download the failed results and run the tests locally or check the log to see why." | tee -a $GITHUB_STEP_SUMMARY >&2
             fi
-            echo ""
-          done
-          if [[ ${FAILED} ]]; then
-            echo "One or more regression tests failed." | tee -a $GITHUB_STEP_SUMMARY
-            echo "Please download the failed results and run the tests locally or check the above log to see why." | tee -a $GITHUB_STEP_SUMMARY
-          fi
 
       - name: Prepare Results for PR Comment
         if : ${{ github.event_name == 'pull_request' }}
         run: |
-          echo "<details>" > summary.txt
-          echo "<summary>Regression Testing Results</summary>" >> summary.txt
-          echo "" >> summary.txt
-          tail -n +1 test/regression-diff/* >> summary.txt
-          echo ""
+          echo "## Regression Testing Results" > summary.txt
+          echo "${{ steps.regression-execution.outputs.stderr }}" >> summary.txt
+          echo "<details>" >> summary.txt
+          echo "<summary>Detailed regression test results.</summary>" >> summary.txt
+          echo "${{ steps.regression-comparison.outputs.stdout }}" >> summary.txt
           echo "</details>" >> summary.txt
```

Comment:
Unless stdout is guaranteed to have a blank line at the end, you will need to add `echo ""` since `</details>` won't work without a blank line before it

Reply:
That's so much nicer - I will be doing that from now on

Reply:
<details>
You can put the summary at the end (or even in the middle!). Helpful when your script spits out things as it goes and doesn't know the conclusion until it is complete
<summary>Something I discovered yesterday when trying to get this to work...</summary>

```
<details>
You can put the summary at the end! helpful when your script spits out things as it goes 
and doesn't know the pass/fail conclusion until it is complete
<summary>Something I discovered yesterday when trying to get this to work...</summary>
</details>
```
</details>